WordPress/WordPressTest/SiteManagementServiceTests.swift

```diff
@@ -115,8 +111,7 @@ class SiteManagementServiceTests: CoreDataTestCase {
         mockRemoteService.successBlockPassedIn?()
         waitForExpectations(timeout: 2, handler: nil)
 
-        let shouldBeRemoved = try? context.existingObject(with: blogObjectID)
-        XCTAssertFalse(shouldBeRemoved != nil, "Blog was not removed")
+        try XCTAssertEqual(context.count(for: Blog.fetchRequest()), 0)
```

Comment:
Changed this assertion here because the blog will exist in the `context` until `context.save()` is called. Using `NSFetchRequest` is a more accurate way to check if the object is deleted from the database.

Reply:
Sounds good 👍 

Do you know whether there are more of this kind of tests that we should consider upgrading (in a dedicated all-in-one PR)?

Reply:
I'm not sure, this is the first time I ran into this kind of issue, maybe I'll find more? I can only notice them if I happen to look at the test case or the test case fails during development. 😄